src/core.c/Setty.rakumod

```diff
@@ -174,9 +174,10 @@ my role Setty does QuantHash {
           nqp::concat(
             nqp::concat(
               nqp::concat(self.^name,'.new('),
-              nqp::join(",",Rakudo::QuantHash.RAW-VALUES-MAP(self, *.raku))
-            ),
-            ')'
+              nqp::join(",", Rakudo::Sorting.MERGESORT-str(
```

Comment:
Feels this slipped in?

Reply:
This is intentional: we need these unordered collections to return an ordered accounting via `.raku`.